src/main/scala/cromwell/engine/backend/jes/JesBackendCall.scala

```diff
@@ -112,4 +114,11 @@ class JesBackendCall(val backend: JesBackend,
 
   override def useCachedCall(avoidedTo: BackendCall)(implicit ec: ExecutionContext): Future[ExecutionHandle] =
     backend.useCachedCall(avoidedTo.asInstanceOf[JesBackendCall], this)
+
+  override val pollBackoff = new ExponentialBackOff.Builder()
+    .setInitialIntervalMillis(30.seconds.toMillis.toInt)
+    .setMaxIntervalMillis(2.minutes.toMillis.toInt)
+    .setMaxElapsedTimeMillis(Integer.MAX_VALUE)
+    .setMultiplier(1.1)
+    .build()
```

Comment:
It'd be nice to have most of this done in the BackendCall.scala and only override `initialIntervalMillis: Int` and `maxInteralMillis: Int` in the subclasses